### ISSUE TITLE
Bolts Swift macOS Compiler Error in Xcode 10

### ISSUE THREAD

**Issue by cliftonlabrum** [BoltsFramework/Bolts-Swift#76]
I just installed Parse via Cocoapods on **Xcode 10**. I'm using this Podfile:

```
def shared
  use_frameworks!
  pod 'Parse'
  pod 'ParseLiveQuery'  
end

target 'MyApp Mac' do
  shared
end

target 'MyApp iOS' do
  shared
end
```
...and my Mac app files to compile with the error:

> Missing argument for parameter #1 in call - Task.swift

Screenshot:
<img width="1155" alt="error" src="https://user-images.githubusercontent.com/2513753/45907371-77660780-bdac-11e8-80f9-92c70305d3d0.png">

I've tried deleting `/DerivedData`, cleaning the build folder, removing Podfile caches and reinstalling, and restarting Xcode.

Any ideas of what else I can try?

**Comment by ShawnBaek** [comment]
I found the solution

Open the Bolt-Swift -> Task.swift -> emptyTask()
Change  .success() ->  .success(())

```swift
 class func emptyTask() -> Task<Void> {
        return Task<Void>(state: .success(()))
    }
```

**Comment by nlutsenko** [comment]
This is fixed as of Bolts-Swift v1.4.0 by #64.

**Comment by holonight** [comment]
executor.execute {
            self.trySet(state: TaskState.fromClosure()
        } 

pls help me fromClosure Missing argument for parameter #1 in call
